### PR TITLE
framework/st_things : Resolves issue where time is reset on reboot

### PR DESCRIPTION
**Pull request by JeongJunSik**
Resolves issue where time is reset on reboot
Solution: Set the system time by taking the time from the NTP server

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check Result:
framework/src/st_things/things_stack/things_stack.c:199: ERROR: [SPC_M_OPR] space required after that ',' (ctx:VxV)

**Comment by sunghan-chang** [comment]
Conflict and coding rule error.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by sunghan-chang** [comment]
Please find 8th on [[commit rule]](Merge branch 'master' into 0115_ntp).

File: framework/src/st_things/Kconfig [hunk 1]
```diff
@@ -17,6 +17,9 @@ config ST_THINGS
 	depends on FS_ROMFS
 	---help---
 		SmartThings Things Management for applications to use and control devices
+	depends on NETUTILS_NTPCLIENT
```
Comment (sunghan-chang) [inline framework/src/st_things/Kconfig]: 1. Your source codes look like not mandatory. It looks option. Why do you set it as ```depends on```?
2. Move this before above ```---help---``` and remove below help.

File: framework/src/st_things/things_stack/things_stack.c [hunk 2]
```diff
@@ -186,6 +190,27 @@ static void *__attribute__((optimize("O0"))) t_things_wifi_join_loop(void *args)
 	return NULL;
 }
 
+#ifdef CONFIG_NETUTILS_NTPCLIENT
```
Comment (sunghan-chang) [inline framework/src/st_things/things_stack/things_stack.c]: If you keep ```depends on```, this conditional is not needed.

**Pull request closed without merging**